python/src/viewer.py

```diff
@@ -553,7 +586,16 @@ def show(self, **kwargs):
         """
         Display the graph.
 
-        See http://matplotlib.org/api/figure_api.html#matplotlib.figure.Figure.show
+        See https://matplotlib.org/api/_as_gen/matplotlib.figure.Figure.html#matplotlib.figure.Figure.show
```

Comment:
I think we should not mention any html link & just refer to matplotlib as previously.
Notice also that we should make sure in this case that links still exists in the case you proposed ...

Reply:
I do not get it. The previous link was broken, and I fixed it. Do you want to remove any link (which is different from the previous situation)?

Reply:
Ok. I remove all links to the Matplotlib website.